students/john_navitsky/session06/test_mailroom.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import mailroom
+import pytest
+import io
+
+
+def test_true():
+    assert True
+
+
+def test_parse_name():
+    assert mailroom.parse_name("Mary Jo Smith, IV") == {
```

Comment:
just a note, if you're curious, for a "Full" name structure:

First, Middle, von, Last, Jr

at least this is how BiBTeX (bibliography management software) handles it --it's pretty universal actually, at least for European names.